config/rbac/role.yaml

```diff
@@ -106,8 +106,12 @@ rules:
   resources:
   - instrumentations
   verbs:
+  - create
```

Comment:
Do we need create/delete permissions for the upgrade?

Reply:
good one, we don't need to create or delete